web/src/views/SubmissionPortal/Components/SubmissionList.vue

```diff
@@ -202,9 +259,69 @@ export default defineComponent({
                 mdi-arrow-right-circle
               </v-icon>
             </v-btn>
+            <v-menu
+              offset-x
+            >
+              <template #activator="{ on }">
+                <v-btn
+                  text
+                  icon
+                  class="ml-1"
+                  v-on="on"
+                >
+                  <v-icon>
+                    mdi-dots-vertical
+                  </v-icon>
+                </v-btn>
+              </template>
+              <v-list>
+                <v-list-item
+                  v-for="(entry, i) in OverFlowMenuItems"
+                  :key="i"
+                  @click="() => handleOverflowMenu(item,entry.title)"
+                >
+                  <v-list-item-title>{{ entry.title }}</v-list-item-title>
+                </v-list-item>
+              </v-list>
+            </v-menu>
           </template>
         </v-data-table>
       </v-card>
     </v-card>
+    <v-row justify="center">
+      <v-dialog
+        v-model="deleteDialog"
+        :width="630"
+        class="ma-5"
+      >
+        <v-card>
+          <v-spacer />
+          <v-card-title class="ma-3 text-h4">
+            Are you sure you want to delete this submission in progress?
```

Comment:
If we keep a ref of which submission triggered the dialog (i.e. `deleteDialogSubmission` in my previous comment, then we can use that to add more contextual information here. That is, it can be something like:

```vue
<p v-if="deleteDialogSubmission">
  <!-- todo: handle case where study_name is blank -->
  This will delete "{{ deleteDialogSubmission.study_name }}" and all associated data.
</p>
<p>This cannot be undone.</p>
```

Reply:
Added the change, and a check to see if deleteDialogSubmission is null before making the dialog visible.